pandas/tests/reshape/test_pivot.py

```diff
@@ -1288,6 +1288,39 @@ def test_pivot_number_of_levels_larger_than_int32(self):
             df.pivot_table(index='ind1', columns='ind2',
                            values='count', aggfunc='count')
 
+    @pytest.mark.parametrize('dropna', [
+        True, False
+    ])
+    def test_pivot_table_aggfunc_dropna(self, dropna):
+        # GH 22159
```

Comment:
do we have a sufficient test when aggfunc is just a scalar? if not can you add

Reply:
@jreback I'm not sure what you mean by this. How can aggfunc be a scalar? Could you please elaborate on this?

Reply:
@alexander-ponomaroff for example when `aggfunc=np.sum`

Reply:
@WillAyd Thank you, it doesn't seem like there is a test with dropna and scalar aggfunc together. So I will add one right now.

Reply:
I added a second test, where `aggfunc=np.mean`. Please take a look and let me know if it's sufficient.